openshift-ironic.spec

```diff
@@ -0,0 +1,293 @@
+%global pyver 3
+%global full_release ironic-%{version}
+
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+
+Name:           openshift-ironic
+Epoch:          1
+Summary:        OpenShift Baremetal Hypervisor API (ironic)
+# (rpittau) we probably need to change this
+Version:        12.1.1
+Release:        1%{?dist}
+License:        ASL 2.0
+URL:            http://www.openstack.org
+# (rpittau) maybe indicate version here ?
+Source0:        ironic.tar.gz
+
+Source1:        openshift-ironic-api.service
+Source2:        openshift-ironic-conductor.service
+Source3:        ironic-rootwrap-sudoers
+Source4:        ironic-dist.conf
+Source5:        ironic.logrotate
+
+BuildArch:      noarch
+BuildRequires:  openstack-macros
+BuildRequires:  python%{pyver}-setuptools
+BuildRequires:  python%{pyver}-devel
+BuildRequires:  python%{pyver}-pbr
+BuildRequires:  openssl-devel
+BuildRequires:  libxml2-devel
+BuildRequires:  libxslt-devel
+BuildRequires:  gmp-devel
+BuildRequires:  python%{pyver}-sphinx
+BuildRequires:  systemd
+# Required to compile translation files
+BuildRequires:  python%{pyver}-babel
+# Required to run unit tests
+BuildRequires:  python%{pyver}-alembic
+BuildRequires:  python%{pyver}-automaton
+BuildRequires:  python%{pyver}-cinderclient
+BuildRequires:  python%{pyver}-eventlet
+BuildRequires:  python%{pyver}-futurist
+BuildRequires:  python%{pyver}-glanceclient
```

Comment:
Side note: since we don't have glance, we should fine a way to make it optional. I'll add a card for that.

Reply:
I think if we remove the check we can just purge all the build requirements related to unit tests

Reply:
Please don't remove the check, it's the only safeguard we have in place.